components/brave_new_tab_ui/api/initialData.ts

```diff
@@ -100,28 +108,6 @@ export async function getInitialData (): Promise<InitialData> {
   }
 }
 
-export async function getBinanceBlackList (): Promise<any> {
```

Comment:
This was no longer needed and moved to the same format as the other widgets. We were originally checking `onlyAnonWallet` as Binance was not allowed in Japan, but JP is part of the blacklist checked by `binance.isSupportedRegion` so not needed now.